proto/metadata.proto

```diff
@@ -49,7 +49,7 @@ message Metadata {
     // The empty string will match any software version.
     string software_version_regex = 4;
     // Reserved field numbers and identifiers.
-    reserved 2;
+    reserved 2, 84;
```

Comment:
Since this field number is for the `Deviations` message the `reserved 84` would go to the deviations message. 

Also, fix the failing  static analysis check

Reply:
fixed, please check